gammapy/data/tests/test_observation_summary.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from astropy.coordinates import Angle, SkyCoord
+
+from gammapy.data import DataStore
+from gammapy.data import ObservationTable
+from gammapy.data import ObservationTableSummary
+
+from ...utils.testing import data_manager, requires_data, requires_dependency
+
+from ...datasets import gammapy_extra
+
+def init_summary():
+    """Init summary table with test gammapy-extra data
+
+    Returns
+    -------
+    summary = `gammapy.data.ObservationTableSummary`
+        Summary table 
+    """
+    
+    datas_tore = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
```

Comment:
There's this error: https://travis-ci.org/gammapy/gammapy/jobs/131172223#L924

You should see that locally as well when you run this test, no?
```
python setup.py test -V -t gammapy/data/tests/test_observation_summary.py
```